.github/workflows/ci.yml

```diff
@@ -1,16 +1,23 @@
-name: Run unit tests & check conventions
```

Comment:
@parhamsaremi why are you deleting this?

Reply:
oops this was a mistake.